### PR TITLE
create Event Page

### PR DESCRIPTION
**Pull request by MarcusSanAntonio**
Screenshot 2024-12-16 at 6 04 53 PM

### CODE REVIEW

File: src/app/events/page.tsx [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+  Text,
+  Group,
+  Container,
+  Title,
+  Alert,
+  Box,
+} from '@mantine/core';
+import { useForm, isNotEmpty } from '@mantine/form';
+
+type EventFormValues = {
```
Comment (Caposto) [inline src/app/events/page.tsx:21]: Just a note, we can probably add a function for users to RSVP to an event. In that case we would just add an users: [] field to the event object

**Review by Caposto — APPROVED** [review]
left some notes but otherwise LGTM

**Comment by Caposto** [comment]
I think there might have been an incorrect rebase since its adding some files that are already there

**Review by RyanCamburn — APPROVED** [review]
🔥 LGTM

**Pull request merged**